src/modules/deps/diff.ts

```diff
@@ -0,0 +1,22 @@
+import chalk from 'chalk'
+import { SessionManager, matchedDepsDiffTable } from 'vtex'
+import { getCleanDependencies } from './utils'
+
+export default async (workspace1 = 'master', workspace2?: string) => {
+  workspace2 = workspace2 ?? SessionManager.getSingleton().workspace
+
+  const deps1 = await getCleanDependencies(workspace1)
+  const deps2 = await getCleanDependencies(workspace2)
+  const diffTable = matchedDepsDiffTable(workspace1, workspace2, deps1, deps2)
+
+  if (diffTable.length === 1) {
+    return console.log(
+      `${chalk.yellow('Dependency diff')} between ${chalk.yellow(workspace1)} and ${chalk.yellow(
```

Comment:
Lets also wrap those messages on our strings file? :D

Reply:
Sure! I didn't change this code (only a few things to fix some errors) in this PR so that it wouldn't have too many changes... But I guess this one more change won't hurt 😝